react_frontend/src/assets/SearchButton.css

```diff
@@ -0,0 +1,17 @@
+.search-button input{
```

Comment:
Similar to the earlier comment we might want to change to relative units later on.

Reply:
I originally went with pixels for the units instead of relative units because I had a difficult time getting the looks of my search button and bar to match with the designs on Figma. I will start to try to use relative units when dealing with styling from now on, so the codebase is more streamlined with our intended designs. 👍

Reply:
Sounds good! It'll be helpful for future components 👍🏾